chromium_src/third_party/ethash/src/lib/keccak/keccak.c

```diff
@@ -7,4 +7,4 @@
 #undef __x86_64__
 #endif
 
-#include "src/lib/keccak/keccak.c"
+#include "../../../third_party/ethash/src/lib/keccak/keccak.c"
```

Comment:
it seems like we should have 7 parents here (```../```) to have root at ```./src/brave```
```
brave/chromium_src/brave/third_party/ethash/src/lib/keccak
    7            6     5           4      3   2   1
```

```#include "../../../../../../../third_party/ethash/src/lib/keccak/keccak.c"```

Reply:
Yep, I would have noticed from CI on Windows but I'm working from a mac. Thanks. Fixed in 8e6f04b3ec681d9478bdf1c2e785a43937730458